ioservice/ut/cob_foms.c

```diff
@@ -864,6 +867,7 @@ static void md_cob_create_delete()
 	md_cob_fop_create_delete_test(true, &CONF_PVER_FID, 0);
 	/* Create the same mdcob again. */
 	md_cob_fop_create_delete_test(true, &CONF_PVER_FID, -EEXIST);
+	md_cob_fop_create_delete_test(false, &CONF_PVER_FID, 0);
```

Comment:
Why is this? Could you please provide more details?

Reply:
This is to delete the cob created with `CONF_PVER_FID` which otherwise was never getting freed.